techlibs/xilinx/abc_xc7.box

```diff
@@ -37,22 +39,15 @@ CARRY4 4 1 10 8
 580 526 507 398 385 508 528 378 380 114
 
 # SLICEM/A6LUT
-# Inputs: A0 A1 A2 A3 A4 D DPRA0 DPRA1 DPRA2 DPRA3 DPRA4 WCLK WE
+# Box to emulate comb/seq behaviour of RAMD{32,64} and SRL{16,32}
```

Comment:
Do you think you could expand this comment a bit, for anyone that comes across this in the future?